components/binary_sensor/mpr121.rst

```diff
@@ -0,0 +1,60 @@
+MPR121 Capacitive Touch Sensor
+==============================
+
+.. seo::
+    :description: Instructions for setting up MPR121 Capacitive Touch Sensor
+    :image: mpr121.png
+    :keywords: MPR121
+
+The ``mpr121`` sensor platform allows you to use your MPR121
+(`datasheet <https://cdn-learn.adafruit.com/downloads/pdf/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial.pdf>`__,
+`Adafruit`_) Capacitive Touch Sensor with esphomelib. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/mpr121.png
```

Comment:
If the two images are exact duplicates you can also just remove the one in `binary_sensor` and use `/images/mpr121.png` here